baybe/utils/dataframe.py

```diff
@@ -733,3 +759,7 @@ def wrapper(df: pd.DataFrame, /) -> pd.DataFrame:
         return wrapper
 
     return decorator
+
+
+class _ValidatedDataFrame(pd.DataFrame):
+    """Wrapper indicating the underlying experimental data was already verified."""
```

Comment:
Really minor, but I think this is the correct terminology (and matches the name)
```suggestion
    """Wrapper indicating the underlying experimental data was already validated."""
```

Reply:
changed